gnssanalysis/gn_io/sp3.py

```diff
@@ -73,116 +91,86 @@ def sp3_clock_nodata_to_nan(
     :return None
     """
     nan_mask = sp3_df[("EST", "CLK")] >= SP3_CLOCK_NODATA_NUMERIC
-    sp3_df.loc[nan_mask, ("EST", "CLK")] = _np.NAN
+    sp3_df.loc[nan_mask, ("EST", "CLK")] = _np.nan
+
 
+def mapparm(old: Tuple[float, float], new: Tuple[float, float]) -> Tuple[float, float]:
+    """
+    Maps values from the old range to the new range.
```

Comment:
Would it be more accurate for this docstring to say this _generates_ scale and offset (mapping parameters) **to enable** remapping. This function isn't for direct data processing.

Reply:
Updated, it wwas the original doc, I just did put it in the same sphinx format than all others...